### PR TITLE
[FIX] point_of_sale: new table duplicates

### PR DESCRIPTION
**Pull request by switch87**
Add new tables in POS screen, close the screen and resume the session
again, the tables you created duplicate

closes https://github.com/odoo/odoo/issues/24831
closes https://github.com/odoo/odoo/issues/25281

task: 1895292

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/pos_restaurant/static/src/js/floors.js [hunk 1]
```diff
@@ -259,7 +259,8 @@ var TableWidget = PosBaseWidget.extend({
                         args: [[['id', '=', table_id]], fields],
                         limit: 1,
                     })
-                    .then(function (table){
+                    .then(function (result){
+                        var table = result[0];
```
Comment (nim-odoo) [inline addons/pos_restaurant/static/src/js/floors.js]: I don't get it. there is a `limit: 1` on the RPC, so what does this do?

Comment (lastalfriday) [reply addons/pos_restaurant/static/src/js/floors.js]: Maybe there was a change in the rpc at some point of time? The result however is an array, but maybe when the code was written it only returned the object. ref: https://github.com/odoo/odoo/issues/27921
selection_110

**Comment by nim-odoo** [comment]
@qle-odoo please read https://github.com/odoo/odoo/pull/27924#discussion_r226248795

**Comment by switch87** [comment]
@nim-odoo can you do your robodoo-magick?

**Pull request closed without merging**